egs2/accented_french_openslr56/asr1/local/remove_missing.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
ditto
it's better to add a comment on how this script works with an example of before and after this proccessing

Reply:
ok